tests/test_fxa_test_account.py

```diff
@@ -5,26 +5,42 @@
 import re
 
 import pytest
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
 
-from fxapom.fxapom import AccountNotFoundException, DEV_URL, FxATestAccount
+from fxapom.fxapom import DEV_URL, FxATestAccount, WebDriverFxA
 
 
-@pytest.mark.skip_selenium
 class TestFxATestAccount(object):
 
+    _fxa_unknown_account_error_locator = (By.CSS_SELECTOR, '#main-content div.error')
+
+    @pytest.mark.skip_selenium
     def test_default_environment_should_be_dev(self):
         account = FxATestAccount()
         assert DEV_URL == account.url
 
+    @pytest.mark.skip_selenium
     def test_create_new_account(self, account):
         assert account.is_verified
         # Test logging in - will throw an exception if log in fails
         account.login()
 
+    @pytest.mark.skip_selenium
     def test_new_account_pw_does_not_contain_numbers(self, dev_account):
         assert re.search(r'\d', dev_account.password) is None
 
-    def test_delete_account(self, account):
-        account.delete_account()
-        with pytest.raises(AccountNotFoundException):
-            account.login()
+    def test_del(self, mozwebqa):
+        """ Check that the __del__ method does destroy the FxA """
+        account = FxATestAccount()
+        saved_account_info = {'email': account.email,
+                              'password': account.password}
```

Comment:
Nit: I don't think we need a dict here, we can just use variables named email and password.